settings.gradle

```diff
@@ -13,6 +13,25 @@ buildscript {
 apply plugin: 'ch.myniva.s3-build-cache'
 apply plugin: 'com.gradle.enterprise'
 
+rootProject.name = 'testcontainers-java'
```

Comment:
Just to be sure - you just moved these lines without changing them, right? Or was it done by accident and we should probably revert it? :)

Reply:
Moved deliberately; I'm not sure why but it feels right to have these elements near the top of the file, and leave the cache/buildscan related parts at the bottom.